app/views/homes/_events.html.erb

```diff
@@ -0,0 +1,56 @@
+<section>
+  <div class="events-container">
+    <div class="upcoming-event-content">
+      <div class="upcoming-event-details">
+        <p class="upcoming-event-title">
+          Our next event
+        </p>
+        <p class="upcoming-event-header">
+          <strong>Meeting</strong> (second Tuesday of every month)
+        </p>
+        <div class="upcoming-event-date">
+          <span>
+            February 3, 2018</br>
+            6:45PM to 10:00PM
+          </span>
+        </div>
+        <div class="upcoming-event-location">
+          <span>
+            ezCater</br>
+            101 Arch St, 15th Floor</br>
+            Boston, MA
+          </span>
+        </div>
+        <div class="upcoming-event__buttons">
+          <%= link_to("RSVP", "#", class: "upcoming-event__button") %>
+          <%= link_to("See all events", "#", class: "upcoming-event__button") %>
+        </div>
+      </div>
+    </div>
+
+    <div class="event-types">
+      <div class="card">
+        <p class="card__title">
+          Project nights
+        </p>
+        <p class="card__header">
+          First tuesday of the month
```

Comment:
We can still capitalize the T in Tuesday though ;)

Reply:
haha, :derp: